docker/ray-ml/Dockerfile

```diff
@@ -23,8 +24,13 @@ RUN sudo apt-get update \
         unzip \
         unrar \
     && $HOME/anaconda3/bin/pip --no-cache-dir install -U pip \
+    # First, install requirements
     && $HOME/anaconda3/bin/pip --no-cache-dir install -U \
            -r requirements.txt \
+    # Then, keep requirements bounds as constraints and install remaining test dependencies
+    && $HOME/anaconda3/bin/pip --use-deprecated=legacy-resolver --no-cache-dir install -U \
+           -c requirements.txt \
+           -r requirements_test.txt \
```

Comment:
Is requirements_test necessary here? Now that we've split up the requirements files, `requirements_test` should only be needed for CI right?

Reply:
Good point actually. I'll remove it